app/assets/javascripts/components/article_finder/article_finder.jsx

```diff
@@ -374,14 +376,25 @@ const ArticleFinder = createReactClass({
       );
     }
 
+    let trackedWikis = [];
```

Comment:
Is there any reason for duplicating this code for generating `trackedWikis` from `course.wikis` across both components that do this, rather than extracting it to `wiki_utils`?

Reply:
none, I'll fix that